platform/android/java/lib/src/org/godotengine/godot/Godot.kt

```diff
@@ -184,6 +185,10 @@ class Godot(private val context: Context) : SensorEventListener {
 			return
 		}
 
+		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
```

Comment:
I don't think you need to check against Android Q since according to the documentation, the night mode apis have been available since api level 8.

Reply:
Removed.